docs/README.md

```diff
@@ -32,6 +32,9 @@
   * [Deleting a user account](#deleting-a-user-account)
 - [Resource aggregation and management](#resource-aggregation-and-management)
   * [Viewing the aggregation service root](#viewing-the-aggregation-service-root)
+  * [Connection methods](#connection-methods)
```

Comment:
no details about conenction method variants

Reply:
Added connection method variant info in the "connection method variant properties" table under "Viewing a single connection method" section.